package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model
+   href="http://download.ros.org/schema/package_format3.xsd"
+   schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+ <name>geographiclib_cmake_module</name>
+ <version>0.0.0</version>
+ <description>Expose CMake targets from older geographiclib libraries for Ubuntu Jammy</description>
+ <maintainer email="25047695+Ryanf55@users.noreply.github.co">Ryan Friedman</maintainer>
+ <license>MIT</license>
+
+ <buildtool_depend>ament_cmake</buildtool_depend>
+ <depend>geographiclib</depend>
+
+ <test_depend>ament_lint_auto</test_depend>
+ <test_depend>ament_lint_common</test_depend>
```

Comment:
If you want to drop these dependencies, you can since they aren't being used.

The main linters that are valuable for this project would be `ament_cmake_lint_cmake` and `ament_cmake_copyright`.